src/components/Inventory/InventoryTable.tsx

```diff
@@ -89,7 +109,7 @@ export default function InventoryTable(props: InventoryTableProps): JSX.Element
         <InventoryFilters filters={filters} setFilters={setFilters} organization={organization} />
       </Grid>
 
-      <Grid xs={12} display='flex' paddingLeft={3} paddingTop={1}>
+      <Grid item xs={12} display='flex' paddingLeft={3} paddingTop={1}>
```

Comment:
There were some errors in the console for using `Grid` without an `item` prop. Added this to silence the error.

Reply:
i believe those console may be tolerable for now - adding an `item` may need a visual check to see if styles need to be updated.
cc'd you on a closed PR (https://github.com/terraware/terraware-web/pull/614)
i have a jira open to look-into/cleanup console errors. https://terraformation.atlassian.net/browse/SW-1819
if in this particular case adding `item` didn't change any of the styling, we should be super fine with the change.

Reply:
Got it — Going forward I'll make sure to check if visual changes are needed any time an `item` prop is added.